lib/puma/dsl.rb

```diff
@@ -539,6 +539,22 @@ def lowlevel_error_handler(obj=nil, &block)
       @options[:lowlevel_error_handler] = obj
     end
 
+    # When force_shutdown_after is set, this allows you to control the message returned
+    # on a force shutdown to the client.
+    #
+    # @example
+    #   force_shutdown_error_response(500, {"Content-Type" => "application/json"}, [JSON.generate({message: "Server shutdown."})])
+    def force_shutdown_error_response(status, headers, response)
+      raise "Headers must be a hash" unless headers.is_a?(Hash)
+      raise "Response must be enumerable" unless response.is_a?(Enumerable)
```

Comment:
Actually, it doesn't have to be an Enumerable. It just has to respond to `each`.